src/templates/partials/tomcat.hbs

```diff
@@ -10,14 +10,14 @@
     port="443"
     SSLEnabled="true">
 
-    <!-- TLS 1.3 requires Java 11 or higher -->
+    {{#if (includes "TLSv1.3" output.protocols)}}<!-- TLSv1.3 requires Java 11 or higher -->{{/if}}
     <SSLHostConfig
 {{#if output.ciphers.length}}
-        ciphers="{{{join output.ciphers ":"}}}"
+        ciphers="{{#if (includes "TLSv1.3" output.protocols)}}{{{join output.cipherSuites ":"}}}:{{/if}}{{{join output.ciphers ":"}}}"
```

Comment:
So this is ugly in a way it always expects output.cipherSuites.length whenever TLSv1.3 is present — otherwise it would emit the ":" delimiter at start.

If that seems fragile I can add extra non-empty check.

Reply:
That's fine.

HandlebarJS is a templating language, not a turing-complete programming language.  We don't have to be too fastidious.  The current guidelines define cipherSuites when TLSv1.3 is defined and I don't see that changing, so I do not consider that fragile.  Related, (for other PRs) the current guidelines list tls_versions in order and I don't see that changing, so we can simplify the code in other templates to use MinVersion = output.protocols[0] to reduce the size of generated code.  We should document these assumptions for posterity, but I do not fear them being broken as guidelines are not updated frequently.  The "current" 5.7 guidelines are over 4 years old.

Reply:
> current guidelines define cipherSuites when TLSv1.3 is defined

My exact assumption, and even when the relation is not set in code, the logic seems to require that for the foreseeable future. ✔️ 

> guidelines list tls_versions in order and I don't see that changing, so we can simplify the code in other templates to use MinVersion = output.protocols[0] 

Agree. Somewhat less self-explanatory when used, but less tedious than iterating over the values awkwardly. ✔️ (I do have a backlog item to even it out across the templates one day… Note to self: maybe _explicitly_ sort the array upon reading in, and document it's to provide the lowest version on that index consistently.)